pom.xml

```diff
@@ -661,6 +663,13 @@
       <plugin>
         <groupId>org.kohsuke</groupId>
         <artifactId>access-modifier-checker</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>enforce</goal>
```

Comment:
Ah, finally [found it](https://github.com/jenkinsci/maven-hpi-plugin/blob/aaf092abf2853410c2101c3e521b515831484dea/src/main/resources/META-INF/plexus/components.xml#L17).